forge-ai/src/main/java/forge/ai/PlayerControllerAi.java

```diff
@@ -1222,19 +1222,19 @@ public String chooseProtectionType(String string, SpellAbility sa, List<String>
     @Override
     public boolean payCostToPreventEffect(Cost cost, SpellAbility sa, boolean alreadyPaid, FCollectionView<Player> allPayers) {
         final Card source = sa.getHostCard();
-        // TODO replace with EmptySa
-        final Ability emptyAbility = new AbilityStatic(source, cost, sa.getTargetRestrictions()) { @Override public void resolve() { } };
-        emptyAbility.setActivatingPlayer(player);
-        emptyAbility.setTriggeringObjects(sa.getTriggeringObjects());
-        emptyAbility.setReplacingObjects(sa.getReplacingObjects());
-        emptyAbility.setTrigger(sa.getTrigger());
-        emptyAbility.setReplacementEffect(sa.getReplacementEffect());
-        emptyAbility.setSVars(sa.getSVars());
-        emptyAbility.setCardState(sa.getCardState());
-        emptyAbility.setXManaCostPaid(sa.getRootAbility().getXManaCostPaid());
-        emptyAbility.setTargets(sa.getTargets().clone());
-
-        if (ComputerUtilCost.willPayUnlessCost(sa, player, cost, alreadyPaid, allPayers)) {
+        if (SpellApiToAi.Converter.get(sa.getApi()).willPayUnlessCost(sa, player, cost, alreadyPaid, allPayers)) {
+            // TODO replace with EmptySa
+            final Ability emptyAbility = new AbilityStatic(source, cost, sa.getTargetRestrictions()) { @Override public void resolve() { } };
+            emptyAbility.setActivatingPlayer(player);
+            emptyAbility.setTriggeringObjects(sa.getTriggeringObjects());
+            emptyAbility.setReplacingObjects(sa.getReplacingObjects());
+            emptyAbility.setTrigger(sa.getTrigger());
+            emptyAbility.setReplacementEffect(sa.getReplacementEffect());
+            emptyAbility.setSVars(sa.getSVars());
+            emptyAbility.setCardState(sa.getCardState());
+            emptyAbility.setXManaCostPaid(sa.getRootAbility().getXManaCostPaid());
+            emptyAbility.setTargets(sa.getTargets().clone());
+
             boolean result = ComputerUtil.playNoStack(player, emptyAbility, getGame(), true); // AI needs something to resolve to pay that cost
```

Comment:
@tool4ever can we find a way for AI to pay the Cost without the Need of an extra empty AbilityStatic ?

Using EmptySa also wouldn't fix the problem.